src/domain/shared/primitives.ts

```diff
@@ -65,3 +65,11 @@ export const paymentAmountFromCents = (cents: UsdCents): UsdPaymentAmount => {
     amount: BigInt(cents),
   }
 }
+
+export const satsFromBtcPaymentAmount = (amount: BtcPaymentAmount): Satoshis => {
+  return Number(amount.amount) as Satoshis
```

Comment:
I guess I did the question to Arvin but why we insist to use bigint if we will cast to number?

Reply:
Removed here https://github.com/GaloyMoney/galoy/pull/1241/commits/290440fe9f5b97352473a0ce76584383961ffbd5